tests/unit/models/ComponentAttribute.spec.js

```diff
@@ -340,6 +347,39 @@ describe('Test class: ComponentAttribute', () => {
 
         componentAttribute.definition.rules.values = null;
       });
+      it('Check type array of object', () => {
```

Comment:
rename to `Check if item in array is valid`
And add test to check `hasError` return false when itemtype are not valid

Reply:
done